dagql/server.go

```diff
@@ -230,47 +237,54 @@ type Loadable interface {
 
 // InstallObject installs the given Object type into the schema.
 func (s *Server) InstallObject(class ObjectType) {
-	s.installLock.Lock()
-	defer s.installLock.Unlock()
+	// FIXME: shortcut to get our "agent" middleware to work
+	// s.installLock.Lock()
+	// defer s.installLock.Unlock()
```

Comment:
:warning: Leaving a comment just to remind us that we need to fix this before merging.